src/Controller/SantaController.php

```diff
@@ -31,13 +32,15 @@ class SantaController extends AbstractController
     private $twig;
     private $logger;
     private $applications;
+    private $statistic;
```

Comment:
`statisticCollector` (Consistency is really important - everywhere, everytime)

Reply:
Done !